hasktorch/src/Torch/Typed/NN.hs

```diff
@@ -260,45 +286,47 @@ data Conv2d (dtype :: D.DType)
 conv2d
   :: forall stride padding
    . _
-  => Conv2d _ _ _ _ _
-  -> Tensor _ _
-  -> Tensor _ _
+  => Conv2d _ _ _ _ _ _
+  -> Tensor _ _ _
+  -> Tensor _ _ _
```

Comment:
PartialTypeSignatures is good on some examples, but I don't know the type definition of API.
If it is not easy  you add the type-definition, could you add some comments for '_' of  'Conv2d _ _ _ _ _ _'?
There is no need to fix it now. 
It ’s good to consider what to do with other PRs.

Reply:
I got strong positive feedback from @apaszke for doing it like this, but we should discuss this approach together as a team, too.
I think the advantage is that the compiler doesn't need the type annotations, and therefore it's an opportunity to reduce boilerplate and noise, to make things less daunting, and to improve readability.
The downside is that one now has to look elsewhere for type hints and type documentation, and reasoning about misbehaviour and compile time errors could be harder.